### ISSUE TITLE
 notebook two throwing stdio.h file not found error on macOS Mojave

### ISSUE THREAD

**Issue by pmaji** [ericmjl/bayesian-stats-modelling-tutorial#50]
Moving this conversation over from [this Twitter thread](https://twitter.com/ByPaulJ/status/1119627287287291904?s=20) to an issue so others can track. 

I'm working through all your nbs to prep for #pycon2019 and ran into an error in the 2nd notebook coming from the pymc3 import. Wanted to let y'all know as I likely won't the the only one to hit this. I've posted a screenshot of the error below, and it appears to be coming from pmc3--specifically the C compiler, from what I can tell from my google searches. 

![Screen Shot 2019-04-20 at 11 40 03 AM](https://user-images.githubusercontent.com/25461122/56459719-c8030a80-6365-11e9-8892-15be70e8ed06.jpg)

P.S. as requested by @ericmjl  -- I did check and this error does NOT appear when using the binder notebooks. So that's a plus!

**Comment by pmaji** [comment]
@ericmjl  / @hugobowne -- following up here on the Twitter thread I just wrote up detailing the remaining error. Looks like a mac-specific problem some folks might have with theano / pmc3. Luckily, it is easily fixed with the terminal commands described at the link below.

https://github.com/frida/frida/issues/338#issuecomment-426777849

I've tested nb's 1-3 after having made these changes, and they all work fine! So closing this issue out for now. See y'all tomorrow!

**Comment by pmaji** [comment]
For the record, @hugobowne is running it w/out any problems on a mac high sierra 10.13.6

**Comment by hugobowne** [comment]
@pmaji is this a different issue to the one you originally raised?

**Comment by pmaji** [comment]
@hugobowne nope! I believe there was some clarification either you or Eric asked for via Twitter and so I recall having a reason for returning to it, but same issue!

**Comment by hugobowne** [comment]
i have no idea what happened on twitter so it would be great to have all details in issue. 

Can you provide the entire error message and the code that threw the error?

Also, for context, why did you re-open after closing it?

**Comment by pmaji** [comment]
Sure thing. So it has been a while and I had to refresh my memory. The reason for the re-opening of the issue after I closed it was that it was a different error message but a similar problem. The night before y'all's presentation I had been trying to run all the nbs, and the initial issue I opened was for the error I first encountered that night. Eventually, I found it was just an issue with my being in the wrong kernel and that fixed everything (I should have posted that update here, apologies for that omission). The next day during the tutorial I ran into a new error--the one described in the newly-opened issue title--but I eventually figured out it had to do with the fact that I was sourcing an older version of one of the packages needed for this. I forget the exact details but I think pymc3 uses theano and I had an out of date version of theano or some other dependency, so after starting a brand new env everything worked as intended. 

tl;dr --> no longer an error that persists on clean startup, my fault for not doing better kernel / env management. (closing)
